samples/sid_end_device/pm_static_nrf5340dk_nrf5340_cpuapp.yml

```diff
@@ -48,34 +48,53 @@ mcuboot_primary_1:
   size: 0x40000
 mcuboot_primary_app:
   address: 0x8200
-  end_address: 0xfc000
+  end_address: 0xf8000
   orig_span: &id002
   - app
   region: flash_primary
-  size: 0xf3e00
+  size: 0xefe00
   span: *id002
 mcuboot_secondary:
   address: 0x0
   device: DT_CHOSEN(nordic_pm_ext_flash)
-  end_address: 0xf4000
+  end_address: 0xf0000
   placement:
     align:
       start: 0x4
   region: external_flash
   share_size:
   - mcuboot_primary
-  size: 0xf4000
+  size: 0xf0000
 mcuboot_secondary_1:
-  address: 0xf4000
+  address: 0xf0000
   device: DT_CHOSEN(nordic_pm_ext_flash)
-  end_address: 0x134000
+  end_address: 0x130000
   region: external_flash
   size: 0x40000
+mfg_fprotect:
+  address: 0xfc000
+  align:
+    start: CONFIG_FPROTECT_BLOCK_SIZE
+  end_address: 0x100000
+  region: flash_primary
+  size: 0x4000
+  span:
+  - mfg_storage_pad
+  - mfg_storage
 mfg_storage:
   address: 0xff000
   end_address: 0x100000
   region: flash_primary
   size: 0x1000
+mfg_storage_pad:
```

Comment:
Form offline discussion - this change makes firmware update on nRF53 breaks device (not a brick, but factory reset needed after dfu).